data/transport missions.txt

```diff
@@ -1975,3 +1975,42 @@ ship "Science Drone"
 		"heat dissipation" 1
 	explode "tiny explosion" 5
 	explode "small explosion" 2
+
+
+
+mission "Loan Broker Reunion"
+	name "Transfer enforcers to <planet>"
+	description "Transport 15 enforcers from <source> to <destination> by <day> to fulfill a request from the New Boston loan broker."
+	deadline
+	passengers 15
+	minor
+	to offer
+		random < 25
+	source "New Boston"
+	destination
+		government "Syndicate"
+	on offer
+		conversation
+			`As you're walking through the spaceport searching for a bar or pub, you notice one person who stands out. A nearly bald, middle-aged man wanders through the crowd in an expensively tailored suit, searching the faces that pass by him. You pause and look at him more closely; he seems oddly familar. Eventually, his gaze settles on you, making eye contact. He blinks a few times in surprise, then begins moving through the crowd towards you.`
+			choice
+				`	(Turn the other way and lose the man in the crowd.)`
+					decline
+				`	(Wait for him to reach you.)`
+				`	(Walk forward to meet him.)`
+			`	When you and the man come face to face, he studies you for a moment. You realize who he is when you hear his voice: "Yes, I thought it was you. It's Captain <last>, correct? I remember you being quite excited to purchase your first ship, even with the outrageously poor interest rate I gave you." It's the broker who lent you the money to purchase your first ship. "No matter," he says. "It seems you've managed."`
```

Comment:
`to purchase your first ship` is repeated twice. I'm not sure the `It's the broker...` sentence is necessary, as it seems implicit at this point who this is by his mention of giving you a loan. Although, I'm not so sure a broker would talk to a past customer by mentioning the "outrageously poor interest rate" they gave them; it just comes off as unnecessarily hostile, almost as if he's mocking you for falling for a bad deal. Perhaps instead he should mention your terrible credit score when you took the loan, and if this mission is changed to offer after you have a better credit score then mention of how terrible it was would show how far you've come.